library/std/src/sync/once_lock.rs

```diff
@@ -196,12 +203,14 @@ impl<T> OnceLock<T> {
         unsafe { self.get_unchecked() }
     }
 
-    /// Sets the contents of this cell to `value`.
+    /// Initializes the cell's value to `value`.
     ///
-    /// May block if another thread is currently attempting to initialize the cell. The cell is
-    /// guaranteed to contain a value when set returns, though not necessarily the one provided.
+    /// May block if another thread is currently attempting to initialize the value of a cell.
+    /// The cell is guaranteed to contain a value when `set` returns, though not necessarily
+    /// the one provided.
     ///
-    /// Returns `Ok(())` if the cell's value was set by this call.
+    /// Returns `Ok(())` if the cell's value was initialized by this call
+    /// and Err(value) if the cell's value was already initialized.
```

Comment:
Backticks around `Err(value)`. Also one of the instances where value->contents would be more accurate.